packages/twenty-server/src/modules/event/standard-objects/timeline-activity.object-metadata.ts

```diff
@@ -87,12 +97,42 @@ export class EventObjectMetadata extends BaseObjectMetadata {
   opportunity: OpportunityObjectMetadata;
 
   @DynamicRelationFieldMetadata((oppositeObjectMetadata) => ({
-    standardId: eventStandardFieldIds.custom,
+    standardId: timelineActivitiestandardFieldIds.custom,
     name: oppositeObjectMetadata.nameSingular,
     label: oppositeObjectMetadata.labelSingular,
     description: `Event ${oppositeObjectMetadata.labelSingular}`,
     joinColumn: `${oppositeObjectMetadata.nameSingular}Id`,
     icon: 'IconBuildingSkyscraper',
   }))
   custom: CustomObjectMetadata;
+
+  @FieldMetadata({
+    standardId: timelineActivitiestandardFieldIds.linkedRecordCachedName,
+    type: FieldMetadataType.TEXT,
+    label: 'Record cached name',
+    description: 'Cached record name',
+    icon: 'IconAbc',
+  })
+  @IsNullable()
+  linkedRecordCachedName: string;
+
+  @FieldMetadata({
+    standardId: timelineActivitiestandardFieldIds.linkedRecordId,
+    type: FieldMetadataType.TEXT,
+    label: 'Record name',
+    description: 'Cached record name',
+    icon: 'IconAbc',
+  })
+  @IsNullable()
+  linkedRecordId: string;
+
+  @FieldMetadata({
+    standardId: timelineActivitiestandardFieldIds.linkedObjectMetadataId,
+    type: FieldMetadataType.TEXT,
+    label: 'Record name',
+    description: 'Cached record name',
+    icon: 'IconAbc',
+  })
+  @IsNullable()
```

Comment:
here too

Reply:
As mentioned previously, since it's a technical field I think it's better to leave it as null than have ''